src/ai/defaultai_seafaring.cc

```diff
@@ -127,46 +127,27 @@ bool DefaultAI::marine_main_decisions(const Time& gametime) {
 	uint16_t ports_count = 0;
 	uint16_t shipyards_count = 0;
 	uint16_t expeditions_in_prep = 0;
+	uint16_t expeditions_ready = 0;
 	uint16_t expeditions_in_progress = 0;
 	bool idle_shipyard_stocked = false;
 
-	// goes over all warehouses (these includes ports)
```

Comment:
this was moved down below the seafaring map check, as doing this check on non seafaring maps makes no sense